doc/man/goto-cc.1

```diff
@@ -96,9 +96,10 @@ files.
 .TP
 \fB\-\-print\-rejected\-preprocessed\-source\fR \fIfile\fR
 Copy failing (preprocessed) source to \fIfile\fR.
-.TP
-\fB\-\-object\-bits\fR \fIN\fR
-Configure the number of bits used for object numbering in CBMC's pointer encoding.
+.SH BACKWARD COMPATIBILITY
+.B goto\-cc
+will warn and ignore the use of \fB\-\-object\-bits\fR, which previous versions
```

Comment:
❓ Where is this warning produced in this PR? Or does that already exist?

Reply:
goto-cc warns about all options that it doesn't explicitly know about, so by virtue of removing it from `gcc_cmdline.cpp` we get this.